sdk/storage/azure-storage-blob/azure/storage/blob/_deserialize.py

```diff
@@ -186,24 +204,32 @@ def parse_tags(generated_tags: Optional[List["BlobTag"]]) -> Union[Dict[str, str
     return None
 
 
-def load_single_xml_node(element: Element, name: str) -> Union[Element, None]:
-    return element.find(name)
+def load_single_xml_node(element: Element, name: str) -> Element:
```

Comment:
Okay, now I know why I changed it 😄 
Line 217 in `load_many_xml_nodes` is:
`element = load_single_xml_node(element, wrapper)`

So it's complaining that element is type-hinted as non-optional type Element _but_ `load_single_xml_node` is type-hinted as `Optional[Element]` (originally Union[Element, None])

Also, originally `def load_many_xml_nodes` defined wrapper as: 
`wrapper: Element = None`

This can't be true as it probably needs to be `Optional[Element]`?

Looking forward to your guidance here since we will need to end up making changes since the old type-hints are incompatible